Rcpp.Rmd

```diff
@@ -641,6 +641,8 @@ LogicalVector is_naC(NumericVector x) {
 is_naC(c(NA, 5.4, 3.2, NA))
 ```
 
+<!-- Is not clear what "sugar function" means. -->
```

Comment:
Oh I think this whole bit can be removed since I removed the large section on sugar functions.

Reply:
I'll change it to 

``Another alternative is the function `is_na()`, which takes a vector and returns a logical vector.``

Reply:
Sorry, I meant remove the entire section that talks about `is_na()`

Reply:
Oh, ok. I've removed everything after 
````
```{r}
str(missing_sampler())
```
````